core/primitives/src/errors.rs

```diff
@@ -65,6 +192,9 @@ pub enum ActionError {
     TriesToUnstake(AccountId),
     TriesToStake(AccountId, Balance, Balance, Balance),
     FunctionCallError(String), // TODO type
+    /// Error occurred if any of the new ActionReceipts created by the FunctionCall action fails the
+    /// validation.
+    NewReceiptValidationError(ReceiptValidationError),
```

Comment:
at the first sight it's not clear why the error variant contains a `Receipt` word (the comment doesn't make it clear..)